python/tests/test_sklearn_vw.py

```diff
@@ -228,96 +263,27 @@ def test_predict(self, data):
         assert np.allclose(raw_model.predict(data.x), model.predict(data.x))
 
-    def test_delete(self):
-        raw_model = VW()
-        del raw_model
-
-
-def test_tovw():
-    x = np.array([[1.2, 3.4, 5.6, 1.0, 10], [7.8, 9.10, 11, 0, 20]])
-    y = np.array([1, -1])
-    w = [1, 2]
-
-    expected = ['1 1 | 0:1.2 1:3.4 2:5.6 3:1 4:10',
-                '-1 2 | 0:7.8 1:9.1 2:11 4:20']
-
-    assert tovw(x=x, y=y, sample_weight=w) == expected
-
-    assert tovw(x=csr_matrix(x), y=y, sample_weight=w) == expected
-
-def test_save_load(tmp_path):
-    train_file = str(tmp_path / "train.model")
-
-    X = [[1, 2], [3, 4], [5, 6], [7, 8]]
-    y = [1, 2, 3, 4]
-
-    model_before = VWRegressor(l=100)
-    model_before.fit(X, y)
-    before_saving = model_before.predict(X)
-
-    model_before.save(train_file)
-
-    model_after = VWRegressor(l=100)
-    model_after.load(train_file)
-    after_loading = model_after.predict(X)
-
-    assert all([a == b for a, b in zip(before_saving, after_loading)])
-
-
-def test_repr():
-
-    model = VW()
-    expected = "VW('convert_labels:False', 'quiet:True', 'sgd:False')"
-    assert expected == model.__repr__()
-
-    model = VWClassifier()
-    expected = "VWClassifier('convert_labels:False', "\
-    "'loss_function:logistic', 'quiet:True', 'sgd:False')"
-    assert expected == model.__repr__()
-
-    model = VWRegressor()
-    expected = "VWRegressor('convert_labels:False', 'quiet:True', 'sgd:False')"
-    assert expected == model.__repr__()
-
-    model = VW(convert_to_vw=False, oaa=3, loss_function='logistic', probabilities=True)
-    expected = "VW('convert_labels:False', 'loss_function:logistic', "\
-    "'oaa:3', 'probabilities:True', 'quiet:True', 'sgd:False')"
-    assert expected == model.__repr__()
-
-
-def test_sgd_param():
+    def test_repr(self):
+        model = VWRegressor()
+        expected = "VWRegressor(convert_labels: True, convert_to_vw: True, passes: 1, quiet: True)"
+        assert expected == model.__repr__()
 
-    model1 = VWRegressor(sgd=True)
-    model2 = VWClassifier(sgd = True)
-    assert model1.get_params()['sgd'] == True
-    assert model2.get_params()['sgd'] == True
 
-
-class TestVWClassifier:
+class TestVWMultiClassifier:
 
     def test_init(self):
         assert isinstance(VWMultiClassifier(), VWMultiClassifier)
 
     def test_predict_proba(self, data):
-        raw_model = VW(probabilities = True, oaa = 2,  loss_function = 'logistic')
-        raw_model.fit(data.x, data.y)
-
-        model = VWMultiClassifier(oaa = 2, loss_function = 'logistic')
+        model = VWMultiClassifier(oaa=2, loss_function='logistic')
         model.fit(data.x, data.y)
-
-        assert np.allclose(raw_model.predict(data.x), model.predict_proba(data.x))
-        # ensure model can make multiple calls to predict
-        assert np.allclose(raw_model.predict(data.x), model.predict_proba(data.x))
+        actual = model.predict_proba(data.x)
+        expected = None
```

Comment:
why expected is `None`?

Reply:
i should have put todo's here, sorry. I removed the comparison against the raw model because I was scared that if something broke this might not catch it. so what I wanted to do is determine what the correct expected values are and hard code them here. I just put None now so the test would fail until it's fixed.